src/GaussianProcessEmulator.jl

```diff
@@ -340,45 +320,38 @@ function predict(gp::GaussianProcess{FT, GPJL}, new_inputs::Array{FT, 2}, transf
 
     # Check if the size of new_inputs is consistent with the GP model's input
     # dimension. 
-    # new_inputs should be of size N_new_inputs x input_dim, so input_dim
-    # has to equal the input dimension of the GP model(s). 
-    input_dim = size(gp.inputs, 2)
-    output_dim = size(gp.data, 2)
-    N_new_inputs = size(new_inputs, 1)
-    err = "GP object and input observations do not have consistent dimensions"
-    size(new_inputs, 2) == input_dim || throw(ArgumentError(err))
+    input_dim, output_dim = size(gp.input_output_pairs, 1)
+    N_new_inputs = size(new_inputs, 2)
```

Comment:
I think now `N_new_inputs` should be renamed to `N_samples`

Reply:
Done